dashboard-ui/src/components/widgets/EnvironmentControl.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright 2024-2025 Andres Morey
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useEffect, useState } from 'react';
+
+import Form from '@kubetail/ui/elements/Form';
+
+import Modal from '@/components/elements/Modal';  
+import { useIsClusterAPIEnabled } from '@/lib/hooks';
+
+type EnvironmentControlWidgetProps = {
+  className?: string;
+};
+
+const EnvironmentControl = ({ className }: EnvironmentControlWidgetProps) => {
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+
+  const [env, setEnv] = useState<string>(() => localStorage.getItem('kubetail:dev:clusterAPIEnabledOverride') || '');
+
+  useEffect(() => {
+    if (env === '') {
+      localStorage.removeItem('kubetail:dev:clusterAPIEnabledOverride');
+    } else {
+      localStorage.setItem('kubetail:dev:clusterAPIEnabledOverride', env);
+    }
+  }, [env]);
+
+  useIsClusterAPIEnabled(null, env);
```

Comment:
What is the `env` argument for? I don't think `useIsClusterAPIEnabled(null, env)` does anything here.